web/styles/message_view_header.css

```diff
@@ -119,4 +119,50 @@
             vertical-align: middle;
         }
     }
+
+    .narrow_description_extended {
```

Comment:
`view-description-extended` would probably be more in line with current naming schemes:
- We use "message view" or "view" instead of "narrow" these days.
- We prefer dashes to underscores in word separators.

Reply:
Yes, it does. Though there's a lot to do overall, and it needs to be done carefully (`git grep narrow web/src/ | wc -l` has 1k matches), a lot of those are for specific purposes.

Probably the main larger thing to come up with a name for in #frontend is the `narrow_state` module, which is fully 20% of those references. I guess I'll start a thread.